sapbtp-operator-charts/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: btp-networkpolicy
+  namespace: {{ .Release.Namespace }}
+spec:
+  podSelector: {}
+  egress:
+    - ports:
+      - protocol: TCP
+        port: 443
+      - protocol: TCP
+        port: 53           # dns
+      - protocol: UDP
+        port: 53           # dns
+      - protocol: TCP
+        port: 8053         # dns
+      - protocol: UDP
+        port: 8053         # dns
+  ingress:
+    - ports:
+      - protocol: TCP
+        port: 9443      # webhook service
+      - protocol: TCP
+        port: 8081     # readiness and liveness probe
+    - from:
+      - podSelector:
+          matchLabels:
+            app.kubernetes.io/name: prometheus
+      ports:
+      - protocol: TCP
+        port: 8443      # prometheus metrics
```

Comment:
8443 is the service port, which might also be used by prometheus.
But I don't think that this chart has a servicemonitor CR, so prometheus isn't used here

Reply:
I will remove prometheus condition